callbacks.tex

```diff
@@ -39,7 +39,11 @@ \section{The Call Stack}\label{s:callbacks-callstack}
 just as we would assign a number:
 
 \begin{minted}{js}
+\\ assign the function oneMore to the variable anotherName 
```

Comment:
comments in Javascript use `//` (forward slashes) rather than `\\` (backslashes)

Reply:
Should be fixed with commit below